### ISSUE TITLE
I would have loved to back your Kickstarter!

### ISSUE THREAD

**Issue by sodabrew** [MarkBryanMilligan/LanternPowerMonitor#24]
I've had this project starred to watch for updates if you were making a new run of PCBs! Unfortunately, there was no update to the GitHub repo during the Kickstarter campaign to notice via GitHub Notifications that something important was afoot!

Do you have plans to run additional boards? Would you use a platform like Indiegogo to collect post-Kickstarter orders? Would you take a PR to link from the README file in the repo over to the subreddit page to increase discoverability of the community there?

https://www.reddit.com/r/LanternPowerMonitor/

https://www.kickstarter.com/projects/markmilligan/lantern-power-monitor

**Comment by MarkBryanMilligan** [comment]
Hi Aaron, sorry about that.  I should have linked reddit to this a long time ago. I pointed reddit to github but not the other way around.  I'll update the readme with the reddit link, but I'll probably leave the kickstarter out since it's all done.

While I'm waiting for parts to come in to fulfill the Kickstarter rewards, I'm setting up an opencart store which will be available from the website ([https://lanternpowermonitor.com](https://lanternpowermonitor.com)).  I've used all of the profits from the kickstarter to order extra inventory and the extra boards and some CTs will be available for purchase sometime in March.
